### ISSUE TITLE
Support multiple interfaces in service

### ISSUE THREAD

**Issue by joshafeinberg** [bnorm/ktor-retrofit#3]
Right now you only support services that have a single super class

https://github.com/bnorm/ktor-retrofit/blob/master/ktor-retrofit/src/main/kotlin/com/bnorm/ktor/retrofit/RetrofitService.kt#L59

Is there a technical reason for this? I have a branch running it fine without that check and as of Retrofit 2.7 it supports this as well.

Example (this currently crashes)

```
interface Network: NetworkOne, NetworkTwo

class Services(
    networkOne: NetworkOne,
    networkTwo: NetworkTwo
) : Network,
    NetworkOneby networkOne,
    NetworkTwoby networkTwo

retrofitService(service = Services(networkOne, networkTwo))
```

**Comment by bnorm** [comment]
No technical reason not to support this. Thanks for the PR, reviewing it now.

**Comment by bnorm** [comment]
Available with latest `0.1.2` version. Thank you so much for your contribution!

**Comment by joshafeinberg** [comment]
Awesome, glad to help. Did you want me to fix #5 as well or did you just catch that while reviewing my PR?

**Comment by bnorm** [comment]
I just caught it while reviewing your PR. Feel free to tackle it if you want, but there was no expectation for you to do so.
